src/ui/types/errors.ts

```diff
@@ -0,0 +1,7 @@
+interface Errors {
+  INCORRECT_FORMAT: string;
+  BELOW_MINIMUM: string;
+  ABOVE_MAXIMUM: string;
+}
```

Comment:
Can we call this `NumberErrorMessage`? More explicit and applies to both non-percentage and percentage type numbers.